apps/web/utils/reply-tracker/enable.ts

```diff
@@ -137,6 +137,15 @@ export async function createToReplyRule(
   emailAccountId: string,
   addDigest: boolean,
 ) {
+  const emailAccount = await prisma.emailAccount.findUnique({
```

Comment:
1. You can just pass in provider. I'd really think before making db calls unnecessarily like this. We're passing provider around all over the app. There's a very high chance the function calling this already has that info.

2. not actually sure how critical it is to pass provider down. i guess it's just to check we don't create a folder rule for gmail users?